src/App.jsx

```
@@ -99,6 +97,9 @@ import PlacementTest from "./pages/placementTest";
 
 // Rewards System
 import RewardsSystem from "@/pages/aurora-site/rewards/page.jsx";
+import GameBoard from "./components/Games/memory-card/game-board";
```

ESLint

[error] 'GameBoard' is defined but never used.

(no-unused-vars)